easybuild/framework/easyconfig/format/format.py

```diff
@@ -62,8 +62,7 @@
     ['name', 'version', 'versionprefix', 'versionsuffix'],
     ['homepage', 'description'],
     ['toolchain', 'toolchainopts'],
-    ['sources', 'source_urls'],
-    ['patches'],
+    ['source_urls', 'sources', 'patches', 'checksums'],
```

Comment:
Hmm, there was a reason why `patches` was separate here...

Does this affect anything else than dumped easyconfigs?

Reply:
I checked, this (`GROUPED_PARAMS`) is only used in the `dump` method, so only affects dumping of easyconfigs